.github/workflows/lib.yml

```diff
@@ -73,16 +73,15 @@ jobs:
       - name: "Check long_description"
         run: "python -m twine check dist/*"
 
-  # TODO: uncomment when klio deps are available on public PyPI
-  # install-dev:
-  #   name: "klio: verify dev env (ubuntu)"
-  #   runs-on: "ubuntu-latest"
-  #   steps:
-  #     - uses: "actions/checkout@v2"
-  #     - uses: "actions/setup-python@v1"
-  #       with:
-  #         python-version: "3.7"
-  #     - name: "Install in dev mode"
-  #       run: "python -m pip install -e .[dev]"
-  #     - name: "Import package"
-  #       run: "python -c 'import klio; print(klio.__version__)'"
\ No newline at end of file
+  install-dev:
+    name: "klio: verify dev env (ubuntu)"
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-python@v1"
+        with:
+          python-version: "3.7"
+      - name: "Install in dev mode"
+        run: "python -m pip install -e .[dev]"
+      - name: "Import package"
+        run: "python -c 'import klio; print(klio.__version__)'"
```

Comment:
just curious, what's a possible scenario that would cause this to fail?

Reply:
It's mostly a sanity check that contributors can setup everything in a clean env. It comes from those who use windows mostly (but I didn't want to add windows as an OS to test on since it's $$$ ¯\\_(ツ)_/¯ ). 

In terms of what can go wrong w this exact line - I've run into [this exact issue](https://hynek.me/articles/testing-packaging/) a few times, and while we have everything setup correctly, I don't want to go thru that embarrassment again (basically packaging up an empty package).